File: src/Moryx.Orders.Endpoints/Models/ReportModel.cs
// Copyright (c) 2026 Phoenix Contact GmbH & Co. KG
// Licensed under the Apache License, Version 2.0

using System.Runtime.Serialization;

namespace Moryx.Orders.Endpoints.Models
{
    [DataContract]
    public class ReportModel
    {
        [DataMember]
        public ConfirmationType ConfirmationType { get; set; }

        [DataMember]
        public string Comment { get; set; }

        [DataMember]
        public int SuccessCount { get; set; }

        [DataMember]
        public int FailureCount { get; set; }

        [DataMember]
        public string UserIdentifier { get; set; }
    }
}

File: src/Moryx.Shifts.Web/app/src/app/api/models/report-model.ts
/*
 * Copyright (c) 2026 Phoenix Contact GmbH & Co. KG
 * Licensed under the Apache License, Version 2.0
*/

/* tslint:disable */
/* eslint-disable */
import { ConfirmationType } from '../models/confirmation-type';
export interface ReportModel {
  comment?: string | null;
  confirmationType?: ConfirmationType;
  failureCount?: number;
  successCount?: number;
  userIdentifier?: string | null;
}

